src/auth/computeclient.ts

```diff
@@ -33,8 +35,9 @@ export class Compute extends OAuth2Client {
    * Retrieve access token from the metadata server.
    * See: https://developers.google.com/compute/docs/authentication
    */
-  constructor() {
-    super();
+  constructor(options?: ComputeOptions) {
+    options = options || {};
+    super({refreshTokenEarlyMillis: options.refreshTokenEarlyMillis});
```

Comment:
I think we should make refreshTokenEarlyMillis optional here.

Reply:
Done.